### PR TITLE
completed css-brochure

### PR DESCRIPTION
**Pull request by rachaelkim831**
brochure screenshot

### CODE REVIEW

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      <option value="3-4 engineers">3-4 Engineers</option>
+      <option value="5-6 enginners">5-6 Engineers</option>
+      <option value="7-8 enginners">7-8 Engineers</option>
+    </select>
+  </div>
+  <div>
+    <label for="submitbutton">
+      <input type="submit" value="SUBMIT" id="submitbutton"  class="submitButton">
+    </label>
+  </div>
```
Comment (scbowler) [inline css-brochure-form/index.html:45]: Inputs for a form should always be wrapped in a `<form>` element

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking great! Just need to a `<form>` element to your form. Let me know if you have any questions.

**Comment by rachaelkim831** [comment]
brochure screenshot

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**